### DISCUSSION TITLE
Ray Reconstruction?

### DISCUSSION THREAD

**Discussion by WhiteZeroX** [sultim-t/xash-rt#203] [Ideas]
nVidia Ray Reconstruction seems to be working wonders for a lot of commercial titles. I wonder what the likelihood of implimenting it into this project is? 

https://www.nvidia.com/en-us/geforce/news/nvidia-dlss-3-5-ray-reconstruction/

**Comment by TuTuc0** [comment]
Thought about it too. HL1 RT denoising could benefit from it. There are some areas that are only indirectly lit by bounced light and currently have quite a RT noise. Occasionally there are also random white "fireflies".
